pages/SavingsForm.js

```diff
@@ -0,0 +1,47 @@
+import AccountForm from './AccountForm';
+
+class SavingsForm extends AccountForm {
+
+    get savingsSavingsButton (){return $('label[for="Savings"]')}
+
+    get moneyMarketSavingsButton (){return $('label[for="Money Market"]')}
+  
+    get savingsNameInput (){return $('#name')}
+  
+    get newSavingsFormSubmitButton () {return $('#newSavingsSubmit')}
+
+    get newSavingsFormResetButton () {return $('#newSavingsReset')}
+
+    get newSavingsErrorText () {return $('#new-account-error-msg')}
+
+    get savingsTitle () { return $('#page-title') }
```

Comment:
I would just rename this `pageTitle` and even place it in BasePage, as apparently every page in the application has it.